### PR TITLE
Post review linguistic fixes

### PR DESCRIPTION
**Pull request by maciekmm**

### CODE REVIEW

**Review by zemiret — CHANGES_REQUESTED** [review]

File: thesis/03-implementation.tex [hunk 1]
```diff
@@ -141,7 +141,7 @@ \subsubsection{Frontend}
...
+Origuide is a web application. Hence, \tech{Javascript} has been an obvious choice. We have briefly considered \tech{Typescript} as well, but have not used it due to its more enforcing nature, which could have potentially slowed down the development process. When it comes to 3D rendering frameworks \tech{Three.js} is one of the most mature and most frequently used libraries for modern rendering targets such as \tech{WebGL}. The decision to use \tech{Material-UI} has been made without much research on other tools, which we later, due to imposed limitations, regretted. As previously mentioned, we have also introduced \tech{React} early in the process to aid us with state management and organize the frontend application in a structurized manner. The tools chosen for code and software quality assurance have been picked based on our familiarity with them.
```
[the remark this answers is not in the document]
Comment (maciekmm) [reply thesis/03-implementation.tex:144]: The problem is, we don't have enough space to fit an explanation. Maybe we should remove the regret part altogether.

File: thesis/01-vision.tex [hunk 2]
```diff
@@ -1,21 +1,21 @@
 \subsection{Problem characteristics}
```
[the remark this answers is not in the document]
Comment (maciekmm) [reply thesis/01-vision.tex:1]: Dictionary crease pattern works for me:  
image

File: thesis/03-implementation.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (zemiret) [reply thesis/03-implementation.tex:144]: Yeah, sure.

File: thesis/01-vision.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (zemiret) [reply thesis/01-vision.tex:1]: Well, yes. But the format is different. Everywhere else it's uppercase `Figure`. I think it's the use autoref that does that.

[the remark this answers is not in the document]
Comment (maciekmm) [reply thesis/01-vision.tex:1]: That's fixed. Thought you had "??" there.

**Review by zemiret — APPROVED** [review]

**Pull request merged**